kuma/dashboards/views.py

```diff
@@ -77,19 +78,26 @@ def revisions(request):
             query_kwargs['created__range'] = [start_date, end_date]
 
         authors_filter = filter_form.cleaned_data['authors']
-        if (not filter_form.cleaned_data['user'] != '' and
+        if ((not filter_form.cleaned_data['user']) and
            authors_filter not in ['', str(RevisionDashboardForm.ALL_AUTHORS)]):
 
-            # The 'Known Authors' group
-            group, created = Group.objects.get_or_create(name="Known Authors")
-            # If the filter is 'Known Authors', then query for the
-            # 'Known Authors' group
-            if authors_filter == str(RevisionDashboardForm.KNOWN_AUTHORS):
-                query_kwargs['creator__groups__pk'] = group.pk
-            # Else query must be 'Unknown Authors', so exclude the
-            # 'Known Authors' group
+            # Get the 'Known Authors' group.
+            if settings.MAINTENANCE_MODE:
+                try:
+                    group = Group.objects.get(name="Known Authors")
+                except Group.DoesNotExist:
+                    group = None
```

Comment:
The ``Group.objects.get_or_create(name="Known Authors")`` was a hack from when Known Authors was a new group and not in production.  I'm leaning toward retiring the auto-creation, and instead ignore the filter if the group isn't created. In other words, make the ``MAINTENANCE_MODE`` solution the read-write solution as well.  If it makes it harder (such as significant test changes), we can do it in a future PR.